### ISSUE TITLE
Add transaction details to tx monitor modals (TxPending, TxError, TxSuccess)

### ISSUE THREAD

**Issue by desfero** [Neufund/platform-frontend#2610]
**Background**
We should refactor our transaction summary modals to extract tx details as a separate component and then use it also inside `TxPending`, `TxError` and `TxSuccess` modals.

**Designs**
Pending: zpl://screen?pid=5a8a92c89c1a166a6a6e8f37&sid=5c6c044456c8a99b363065c1
External Pending: zpl://screen?pid=5a8a92c89c1a166a6a6e8f37&sid=5c6e8dbffad3a61a6c722cff
TxError: zpl://screen?pid=5a8a92c89c1a166a6a6e8f37&sid=5c6c04467fee003c05ccc6d8
TxPending: zpl://screen?pid=5a8a92c89c1a166a6a6e8f37&sid=5c6c0446386ad09b283d4471

**Acceptance criteria**
Given new transaction when user accept summary then the same summary table should be shown in `TxPending`, `TxError` and `TxSuccess` modals.

**Technical notes**

**Testing notes**
This would be really time-consuming to test as all possible transaction on the platform should be manually checked.

**Comment by glupiawymowka** [comment]
- [ ] for unsuccessful transactions, the link to the support should be "https://support.neufund.org/support/home"  the current one is not working

- [ ] It should be visible for the user, that there is a pending transaction, also when he closes the "your transaction is pending" window.  He should still be blocked from starting any other transaction. 

I've made an "accept payout" transaction

![image](https://user-images.githubusercontent.com/18315589/55074572-ee989300-5090-11e9-8009-77d1d476d615.png)

while it was pending, I made an "redistribute payout" transaction for the same amount.

![image](https://user-images.githubusercontent.com/18315589/55074650-2273b880-5091-11e9-95d8-984dc4e1e8fd.png)

all of the transactions are visible in my Metamask 

![image](https://user-images.githubusercontent.com/18315589/55074786-826a5f00-5091-11e9-9f73-21b6263124af.png)

flows that were tested for correct amounts and titles: 
- [x] withdraw (succes, pending, failed)
- [x] payout (ETH, nEUR | accept, redistribute | success, fail) -> amounts are incorrectly rounded up in the whole flow, ticket to be created. Otherwise it's fine
- [x] investment (success, failed, pending |nEUR & ETH)
- [x] redeem (succes/failed)
- [x] whitelisting (investor, issuer | success, failing) 
- [x] claim tokens (success, failing)
